electron/main.ts

```diff
@@ -58,6 +61,18 @@ ipcMain.on('run-helm', (event, args: any) => {
   }
 });
 
+ipcMain.on('app-version', event => {
+  event.sender.send('app-version', {version: app.getVersion()});
+});
+
+ipcMain.on('check-update-available', () => {
+  autoUpdater.checkForUpdatesAndNotify();
```

Comment:
how/where does the autoUpdater actually check for a new version?

Reply:
It checks our Github releases. If there is a new release It notifies the application and downloads it.

Reply:
ok thanks - is this configurable? i.e. can we tell it check a specific URL? If so - what do we have to host at that URL?

Reply:
Yeah, it is configurable. All we need to do is to place our files on the server. Supported providers are `GitHub Releases`, `Amazon S3`, `DigitalOcean Spaces`, `Bintray`. We can also use our custom server. But I think after Github Releases the best option is AWS S3 with some authentication gate. It works well with private Github repos as well. We just need to set Github Access Token to our environments.

Reply:
ok thanks - we can start with GitHub for now - what I'm after is getting analytics on how often this is checked - as an initial indicator of monokle usage.. i'll create a separate ticket for moving this somewhere else, there might be other aspects we need to be mindful of...

Reply:
We can track this event on the client side. No server-side implementation is needed for this purpose I think. Even google analytics will do what we want. We can get information per user.